src/routes/docs/tutorials/apple/step-7/+page.markdoc

```diff
@@ -0,0 +1,379 @@
+---
+layout: tutorial
+title: Habit UI
+description: Add, update and query Database in SwiftUI
+step: 7
+---
+
+# HomeView UI {% #homeview-ui %}
+
+Using ``TabView`` user will be able to switch between multiple tab from home screen to progress to setting/preference. And with ``blur`` modifier we able to blur the home view and show modal of medium size for user to update the habit goal completed. Also performing async task inside ``task`` modifier in order to fetch habits record
```

Comment:
The code tags aren't working.

also which file are we in, where are we, I'm a little lost.